### ISSUE TITLE
Upload Test Scripts to ALM Test Plan 

### ISSUE THREAD

**Issue by nawabhussain** [sumeet-kushwah/ALM_OTA_Wrapper#1]
I have been looking into uploading scripts to Test Script section in Test Plan. Is it possible ?

**Comment by sumeet-kushwah** [comment]
Yes, its possible, you need to create the test cases, its defined under test.cs

https://github.com/sumeet-kushwah/ALM_OTA_Wrapper/blob/master/ALM_Wrapper/Test.cs

You need to call create function and then adddesignsteps function. Examples are available under

https://github.com/sumeet-kushwah/ALM_OTA_Wrapper/blob/master/ALM_Wrapper_Tests/ALM_Wrapper_Test.cs

Please let me know about any other questions

Thanks,
Sumeet Kushwah
